configs/components/augeas.rb

```diff
@@ -120,7 +122,14 @@
   end
 
   # fix libtool linking on big sur
-  pkg.configure { ["/usr/local/bin/autoreconf --force --install"] } if platform.is_macos?
+  if platform.is_macos?
+    if(platform.os_version.to_i >= 13 && platform.architecture == 'arm64')
```

Comment:
nit, we generally don't parenthesize unless absolutely needed:

```suggestion
    if platform.os_version.to_i >= 13 && platform.architecture == 'arm64'
```